packages/toolpad-app/sentry.client.config.js

```diff
@@ -0,0 +1,17 @@
+// This file configures the initialization of Sentry on the browser.
+// The config you add here will be used whenever a page is visited.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from '@sentry/nextjs';
+
+const SENTRY_DSN = process.env.SENTRY_DSN || process.env.NEXT_PUBLIC_SENTRY_DSN;
```

Comment:
```suggestion
const SENTRY_DSN = process.env.SENTRY_DSN;
```

Reply:
What's the right way to have a client-side readable environment variable in our project, if it's not a build-time variable? Is it using the runtime config, like I've changed it to now?